MetadataExtractor/Formats/Photoshop/DuckyReader.cs

```diff
@@ -9,14 +9,15 @@ namespace MetadataExtractor.Formats.Photoshop
     public sealed class DuckyReader : IJpegSegmentMetadataReader
     {
         public const string JpegSegmentPreamble = "Ducky";
+        internal static ReadOnlySpan<byte> JpegSegmentPreambleBytes => "Ducky"u8;
 
-        ICollection<JpegSegmentType> IJpegSegmentMetadataReader.SegmentTypes { get; } = new[] { JpegSegmentType.AppC };
+        ICollection<JpegSegmentType> IJpegSegmentMetadataReader.SegmentTypes { get; } = [JpegSegmentType.AppC];
```

Comment:
We should bump these to IReadOnlyCollection. Can be a future PR.

Reply:
Annoyingly, HashSet<JpegSegmentType> doesn't implement IReadOnlyCollection, which we use in JpegReader.

Reply:
Looking at usages of the property, they're all enumerations. So `IReadOnlyCollection` or `IEnumerable` are both candidates, and `JpegReader` would be changed to no longer use `HashSet<T>`.